.changeset/angry-timers-press.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap-cloud-sdk/connectivity': patch
+'@sap-cloud-sdk/mail-client': patch
+---
+
+[Fixed Issues] Set `_readableListening` property of Socket to `true` to allow nodemailer to receive SMTP greeting messages. Fixes `Greeting never received` error when sending emails to onPrem mail servers.
```

Comment:
[req] I would also mention one item for reflecting the error `The proxy configuration is undefined`